ocs_ci/ocs/machine.py

```diff
@@ -275,6 +298,39 @@ def create_custom_machineset(
         raise UnsupportedPlatformError("Functionality not supported in UPI")
 
 
+def create_infra_nodes(num_nodes):
+    """
+    Create infra node instances
+
+    Args:
+        num_nodes (int): Number of instances to be created
+
+    Returns:
+        list: list of instance names
+
+    """
+    ms_names = []
+    zone_list = []
+    machinesets_obj = OCP(
+        kind=constants.MACHINESETS, namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE
+    )
+    for machine in machinesets_obj.get()['items']:
+        aws_zone = machine.get('spec').get('template').get('spec').get(
+            'providerSpec').get('value').get('placement').get('availabilityZone')
+        zone_list.append(aws_zone[-1])
+
+    ms_names.extend(
+        [
+            create_custom_machineset(
+                role='infra',
```

Comment:
don't we need to pass label here, as the default label i see in the function is "app-scale"

Reply:
The label necessary for 'infra' nodes has been handled as an extra 'if' condition, assuming that these are extra labels along with whatever default (app-scale) we want.

Don't we need those default labels ? I am not really aware of label requirements here so I request you to clarify on this.

Reply:
AFAIK, app-scale label is a custom label which scale team created for running their scale pgsql test.
@ramkiperiy @tiffanyn108 Please correct me if I wrong.
This is not need for normal deployments (non-scale).

Reply:
I commented on other place that this should not be default behavior of the function and should not be templated by default by this custom label.